### ISSUE TITLE
`react-transform-webpack-hmr` is deprecated and new config

### ISSUE THREAD

**Issue by bsr203** [vslinko/ripster#11]
`react-transform-webpack-hmr` is deprecated to `react-transform-hmr` .

new `.babelrc`

```
{
  "stage": 0,
  "env": {
    "development": {
      "plugins": [
        "react-transform"
      ],
      "extra": {
        "react-transform": {
          "transforms": [{
            "transform": "react-transform-hmr",
            "imports": ["react"],
            "locals": ["module"]
          }, {
            "transform": "react-transform-catch-errors",
            "imports": ["react", "redbox-react"]
          }]
        },
      }
    }
  }
}

```

**Comment by vslinko** [comment]
Could you make a PR with those changes?

**Comment by bsr203** [comment]
will.. try (never did before :-) )... gimme some time as I am heading out.. thanks..

**Comment by vslinko** [comment]
Closed by #12
